### PR TITLE
CE-3478 Add question to infobox

### PR DESCRIPTION
**Pull request by kamilkoterba**
**This code is to be removed after experiment** ~2-3weeks

## Links

* Feature ticket https://wikia-inc.atlassian.net/browse/CE-3473

## Description

This is experiment that adds predefined questions to infoboxes on some pages.
First subtask done for now: [Insert question field to infobox](https://wikia-inc.atlassian.net/browse/CE-3478) 

## Reviewers

@Wikia/community-engineering, @Wikia/x-wing

### CODE REVIEW

File: front/main/app/styles/module/article/_portable-infobox-question.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	&.collapsed {
+		bottom: 45px;
+		position: absolute;
+		z-index: $z-portable-infobox-collapse + 1;
+
+		&::before {
+			background: linear-gradient(to top, $color-gray-light 5%, transparentize($color-gray-light, 1));
+			content: '';
+			height: 45px;
+			left :0;
```
Comment (Grunny) [inline front/main/app/styles/module/article/_portable-infobox-question.scss]: Spacing.

File: front/main/app/styles/module/article/_portable-infobox-question.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	}
+}
+.portable-infobox-question__input[type="text"] {
+	background: transparent;
+	border-bottom: 1px solid $color-blue-gray;
+	box-shadow: none;
+
+	&:focus {
+		border-bottom: 1px solid $color-blue-light;
+		background: transparent;
```
Comment (Grunny) [inline front/main/app/styles/module/article/_portable-infobox-question.scss]: Should be alphabetised. Make sure you run scss-lint to catch these minor coding convention issues, or just add our [githooks](https://github.com/Wikia/guidelines/tree/master/githooks) to catch them on commit.

**Comment by Grunny** [comment]
Minor coding conventions comments, otherwise LGTM as a start to the questions module.

File: front/main/app/components/portable-infobox-question.js [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+			}
+
+			return experimentWikiPage;
+		}),
+		hideInfoboxField(field, value, separator = ',') {
+			Ember.$('.portable-infobox').find('h3').filter((index, elem) => {
+				if (elem.textContent === field) {
+					if (value) {
+						const $valueNode = Ember.$(elem).next('.pi-data-value'),
+							values = $valueNode.html().split(separator).filter((currentValue) => {
```
Comment (hakubo) [inline front/main/app/components/portable-infobox-question.js]: ```$valueNode.html().split(separator).filter((currentValue) => currentValue.indexOf(value) === -1});```

File: front/main/app/mixins/user-feedback-storage.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
+import Ember from 'ember';
+import {getDomain} from '../utils/domain';
+
+export default Ember.Mixin.create({
```
Comment (hakubo) [inline front/main/app/mixins/user-feedback-storage.js]: this sounds more like a model to me. 
What are the reasons for this to be a mixin?

**Comment by hakubo** [comment]
one minor thing I would consider

also the mixin look more like a model

both of these are not blockers
lgtm for a test

File: front/main/app/mixins/user-feedback-storage.js (hunk 4 above)
Comment (rogatty) [inline front/main/app/mixins/user-feedback-storage.js]: +1

File: front/main/app/components/wikia-ui-components/text-field.js [hunk 5]
```diff
@@ -10,12 +10,15 @@
  *      class='additional-custom-class'
  *      onFocusHandler=onFocusHandlerAction
  *      onBlurHandler=onBlurHandlerAction
+ *      placeholder='placeholder text'
+ *      transparent=false
```
Comment (rogatty) [inline front/main/app/components/wikia-ui-components/text-field.js]: We already have `class='additional-custom-class'` which can be used exactly for what you need.

**Comment by rogatty** [comment]
One comment and it'll LGTM as a temporary experiment.

File: front/main/app/mixins/user-feedback-storage.js (hunk 4 above)
Comment (kamilkoterba) [inline front/main/app/mixins/user-feedback-storage.js]: Makes sense. We'll do it this way next time!

**Pull request merged**